src/ports/linux/pnal.c

```diff
@@ -97,6 +97,7 @@ int pnal_save_file (
 
    /* Close file */
    fclose (outputfile);
+   sync();
```

Comment:
Maybe fsync() is better. Syncing/flushing whole file system should probably be avoided.
I think we should rerun test with fsync to check if that works and also compare the timing of  the two

Reply:
Ok, I will try.

Reply:
The `fsync()` approach did unfortunately not work, but I removed the `sync()` from `pnal_clear_file()`.

Reply:
OK

Reply:
It might be worth investigating O_SYNC | O_DIRECT.

Reply:
`fflush()` and `fsync()` together did the trick.